packages/kit/test/apps/basics/test/test.js

```diff
@@ -463,6 +463,12 @@ test.describe.parallel('Shadowed pages', () => {
 
 		expect(headers.head).toEqual(headers.get);
 	});
+
+	test.only('Works with missing get handler', async ({ page, clicknav }) => {
```

Comment:
```suggestion
	test('Works with missing get handler', async ({ page, clicknav }) => {
```

Is there some way to make .only fail in CI like we have in the core Svelte repo?

Reply:
opened https://github.com/sveltejs/kit/pull/3922